### ISSUE TITLE
Add helmetjs

### ISSUE THREAD

**Issue by klouvas** [Workable/orka#140]
https://helmetjs.github.io/

**Comment by alkiskal** [comment]
we are using koa2 so I guess this https://github.com/venables/koa-helmet would be more appropriate. Moreover, is this really needed? What will it provide exactly?

**Comment by klouvas** [comment]
yes we would use the `koa-helmet`. It provides some security features. Feel free to close this issue if it seems that we will not use this package.
